src/manifest.json

```diff
@@ -2,7 +2,7 @@
   "manifest_version": 2,
   "name": "@EXTENSION_NAME@",
   "description": "Urgent post-release fixes for web compatibility.",
-  "version": "18.0.0",
+  "version": "18.0.1",
```

Comment:
One *minor* nit: https://github.com/mozilla-extensions/webcompat-addon/wiki/Versioning-Scheme - that should probably be `18.1.0` - not that it really matters, but let's be consistent. :)